Changes in for.md

saujanya01 opened the pull request.

saujanya01 committed:
Changes in for.md

md-rehman commented:
This updated example looks more appropriate. Do you know why it's not merged?

And @ericwindmill, I also want to know if you are accepting PRs. If yes, I would like to add more examples as well.

saujanya01 commented:
@MD-REHMAN i don't think this repo is active any more

ImgBotApp committed:
[ImgBot] Optimize images

*Total -- 2,908.22kb -> 1,338.08kb (53.99%)

/static/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/logos/logo-48.png -- 16.02kb -> 3.32kb (79.27%)
/static/logos/logo-48.png -- 16.02kb -> 3.32kb (79.27%)
/static/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/logos/logo-48.png -- 16.02kb -> 3.32kb (79.27%)
/static/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/logos/logo-1024.png -- 56.56kb -> 20.04kb (64.56%)
/static/logos/logo-1024.png -- 56.56kb -> 20.04kb (64.56%)
/static/gatsby-advanced-starter/logos/logo-1024.png -- 56.56kb -> 20.04kb (64.56%)
/static/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/logos/logo-1024.png -- 56.56kb -> 20.04kb (64.56%)
/static/screen-shot-v0-1.png -- 237.60kb -> 104.78kb (55.9%)
/static/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/screen-shot-v0-1.png -- 237.60kb -> 104.78kb (55.9%)
/static/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/screen-shot-v0-1.png -- 237.60kb -> 104.78kb (55.9%)
/static/gatsby-advanced-starter/screen-shot-v0-1.png -- 237.60kb -> 104.78kb (55.9%)
/static/screen-shot-v0-2.png -- 420.88kb -> 207.21kb (50.77%)
/static/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/gatsby-advanced-starter/screen-shot-v0-2.png -- 420.88kb -> 207.21kb (50.77%)
/static/gatsby-advanced-starter/screen-shot-v0-2.png -- 420.88kb -> 207.21kb (50.77%)
/static/gatsby-advanced-starter/gatsby-advanced-starter/screen-shot-v0-2.png -- 420.88kb -> 207.21kb (50.77%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>

saujanya01 committed:
Merge pull request #1 from saujanya01/imgbot

[ImgBot] Optimize images
